src/pretix/base/models/orders.py

```diff
@@ -299,6 +299,11 @@ class Order(LockModel, LoggedModel):
         verbose_name=_("Meta information"),
         null=True, blank=True
     )
+    api_meta_info = models.JSONField(
+        verbose_name=_("API-Meta information"),
+        null=True, blank=True,
```

Comment:
Why should this be nullable? Do we need the semantic difference between NULL and {}?

```suggestion
        null=False, blank=True,
```